src/main/controllers/helpers/ApplicationDetailsHelper.ts

```diff
@@ -180,3 +188,180 @@ export const getTseApplicationDecisionDetails = (
   }
   return tseApplicationDecisionDetails;
 };
+
+export const getAllResponses = async (
+  respondCollection: TseRespondTypeItem[],
+  translations: AnyRecord,
+  accessToken: string,
+  res: Response
+): Promise<any> => {
+  const allResponses = [];
+
+  if (respondCollection) {
+    for (const response of respondCollection) {
+      if (response.value.from === Applicant.RESPONDENT || response.value.from === Applicant.CLAIMANT) {
```

Comment:
it's either these or admin so you could do a switch and throw illegal state exception if not one of the three, not too bothered though, up to you.